prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/CO/co-phd.yml

```diff
@@ -0,0 +1,224 @@
+- name: "co-phd"
+  description: "Colorado Department of Public Health and Environment"
+  jurisdiction: "STATE"
+  stateCode: "CO"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+  - name: "full-elr-secondary"
+    organizationName: "co-phd"
+    topic: "full-elr"
+    customerStatus: "active"
+    translation: !<HL7>
+      schemaName: "classpath:/metadata/hl7_mapping/receivers/STLTs/CO/CO-receiver-transform.yml"
+      useTestProcessingMode: false
+      useBatchHeaders: true
+      receivingApplicationName: "CO-ELR"
+      receivingApplicationOID: null
+      receivingFacilityName: "CO"
+      receivingFacilityOID: null
+      messageProfileId: null
+      replaceValue: {}
+      replaceValueAwithB:
+        MSH-6:
```

Comment:
I think, we don't need these replaceValueAwithB any longer for UP.

Reply:
Removed it